infrastructure/docker/services/builder/Dockerfile

```diff
@@ -2,14 +2,13 @@ ARG PROJECT_NAME
 
 FROM ${PROJECT_NAME}_php-base
 
-ARG NODEJS_VERSION=14.x
-RUN echo "deb https://deb.nodesource.com/node_${NODEJS_VERSION} bullseye main" > /etc/apt/sources.list.d/nodejs.list \
-    && apt-key adv --fetch-keys https://deb.nodesource.com/gpgkey/nodesource.gpg.key
+ARG NODEJS_VERSION=16.x
+RUN curl -s https://deb.nodesource.com/gpgkey/nodesource.gpg.key | gpg --dearmor | tee /usr/share/keyrings/nodesource.gpg >/dev/null \
```

Comment:
I think this could be simpler : 

```
curl -s https://deb.nodesource.com/gpgkey/nodesource.gpg.key| gpg --dearmor  > /usr/share/keyrings/nodesource.gpg
```

---

I have another question, why did you change the way the package signature is checked?

Reply:
I will simplify the command, good catch (I followed the way its done by NodeJS scripts)

apt-key will be removed in future distributions versions (see https://wiki.debian.org/DebianRepository/UseThirdParty)

Reply:
Oh, thanks for this Information. I didn't know.

Reply:
Simplified commands 🙂